docs/user-guide/features.rst

```diff
@@ -33,6 +33,20 @@ Administrators can also :ref:`upload <create-errata-recipe>` their own errata to
 a repository. Please see the :doc:`recipes` documentation to learn how to
 perform these operations.
 
+Modularity
+^^^^^^^^^^
+
+Pulp supports modularity_ content use cases up to the `repository mirroring
```

Comment:
i would suggest to explain here in docs( and not forcing the user to open and read the issue)in more details what do you mean by `repository mirroring` - as you already mentioned below that metadata is not processed maybe would be good to highlight also in which format/content type we store it in pulp rpm repo.